EasyProject/src/main/java/org/example/task1_case3/Matrix.java

```diff
@@ -0,0 +1,146 @@
+package org.example.task1_case3;
+import java.util.*;
+
+public class Matrix {
+
+    private final char[][] matrix;
+    private final int col;
+    private final int row;
+
+    public int getCol() {
+        return col;
+    }
+
+    public int getRow() {
+        return row;
+    }
+
+    public Matrix(char[][] m, int col ,int row) {
+        matrix= new char[col][row];
+        for (int i = 0; i < col; i++) {
+            System.arraycopy(m[i], 0, matrix[i], 0, row);
+
+        }
+        this.col=col;
+        this.row=row;
+    }
+
+    public Matrix(int col , int row){
+        matrix = new char[col][row];
+        this.col = col;
+        this.row = row;
+    }
+
+
+
+    public void playTicTacToe(){
+        inputCommand();
+    }
+    public void inputCommand(){
+        Scanner scan = new Scanner(System.in);
+        System.out.print("Input command:");
+        String option = scan.nextLine();
+
+        if ("start easy easy".trim().equals(option.trim()) ) {
```

Comment:
please use CASE instead of if else statements

Reply:
done